mqt/benchviewer/main.py

```diff
@@ -123,9 +142,19 @@ def get_num_benchmarks():
         return jsonify(data)
 
 
-def main():
-    app.run(debug=True)
+def start_server(
+    skip_question=False,
+    activate_logging=False,
+    target_location: str = "mqt/benchviewer/static/files/qasm_output",
+):
+    init(
+        skip_question=skip_question,
+        activate_logging=activate_logging,
+        target_location=target_location,
+    )
+    print("Server is hosted at: ", "http://127.0.0.1:5000" + PREFIX)
```

Comment:
I am not too familiar with Python itself in this regard, but could we make the link (http://127.0.0.1:5000) clickable in the printed statement? Is there any way of using markdown here?

Second question: is it really `http` and not `https`?

Third question: Should this message also contain some information on how to stop the server (ctrl+C or something like that)?

I believe jupyter has all of those features if it helps as a point of reference.

Reply:
![image](https://user-images.githubusercontent.com/23012802/185862093-cd3473d3-b4dd-4f28-9ce4-50c3c6ca6417.png)

Answer to question 1): It is clickable by default, see attached screenshot.

Reply:
To answer the remaining two questions:
2) For the local case it is http, for the actual webserver it is of course https.
3) I added that information.